instrumentation/opentelemetry-instrumentation-pymemcache/src/opentelemetry/instrumentation/pymemcache/__init__.py

```diff
@@ -89,27 +99,29 @@ def _set_connection_attributes(span, instance):
 def _with_tracer_wrapper(func):
     """Helper for providing tracer for wrapper functions."""
 
-    def _with_tracer(tracer, cmd):
+    def _with_tracer(tracer, cmd, sanitize_query):
         def wrapper(wrapped, instance, args, kwargs):
             # prevent double wrapping
             if hasattr(wrapped, "__wrapped__"):
                 return wrapped(*args, **kwargs)
 
-            return func(tracer, cmd, wrapped, instance, args, kwargs)
+            return func(
+                tracer, cmd, sanitize_query, wrapped, instance, args, kwargs
+            )
 
         return wrapper
 
     return _with_tracer
 
 
 @_with_tracer_wrapper
-def _wrap_cmd(tracer, cmd, wrapped, instance, args, kwargs):
+def _wrap_cmd(tracer, cmd, sanitize_query, wrapped, instance, args, kwargs):
     with tracer.start_as_current_span(
         cmd, kind=SpanKind.CLIENT, attributes={}
     ) as span:
         try:
             if span.is_recording():
-                if not args:
+                if not args or sanitize_query:
```

Comment:
I don't think this is the approach we follow elsewhere. The values get replaced with something like `?` or similar.